backend/open_webui/retrieval/utils.py

```diff
@@ -85,7 +79,9 @@ def query_doc(
         )
 
         if result:
-            log.info(f"query_doc:result {result.ids} {result.metadatas}")
+            log.debug(
+                f"get_doc: Found {len(result.ids[0]) if result.ids else 0} results"
```

Comment:
Why do we change the original log content? Is it necessary to change log level to `debug`?